librosa/core/time_frequency.py

```diff
@@ -900,6 +902,70 @@ def octs_to_hz(octs, tuning=0.0, bins_per_octave=12):
     return (float(A440) / 16)*(2.0**np.asanyarray(octs))
 
 
+def A4_to_tuning(A, bins_per_octave=12):
+    """Convert reference frequencies to cents (bin fractions).
+
+    Examples
+    --------
+    >>> librosa.A4_to_tuning()
+    0.
+    >>> librosa.A4_to_tuning(432.0)
+    -0.318
+    >>> librosa.A4_to_tuning([440.0, 444.0], 24)
+    array([   0.,   0.313   ])
```

Comment:
In the examples, it's helpful to have a comment before each one explaining what it does.  [This one](https://librosa.github.io/librosa/generated/librosa.core.tone.html#librosa.core.tone) gives an example of what I mean.